cli/sql.go

```diff
@@ -67,11 +133,21 @@ func runInteractive(db *sql.DB, dbURL string) (exitErr error) {
 	if len(fullPrompt) == 0 {
 		fullPrompt = " "
 	}
-	continuePrompt := strings.Repeat(" ", len(fullPrompt)-1) + "-"
+
+	continuePrompt = strings.Repeat(" ", len(fullPrompt)-1) + "-"
 
 	fullPrompt += "> "
 	continuePrompt += "> "
 
+	return fullPrompt, continuePrompt
+}
+
+// runInteractive runs the SQL client interactively, presenting
+// a prompt to the user for each statement.
+func runInteractive(db *sql.DB, dbURL string) (exitErr error) {
+
```

Comment:
Oh really I didn't know this was a style difference. I always do this in functions I write; I guess I didn't realize there was a difference. Should I patch it?